tests/system_tests_console.py

```diff
@@ -51,7 +51,7 @@ def is_cmd(name):
             devnull = open(os.devnull, "w")
             subprocess.Popen([name], stdout=devnull, stderr=devnull).communicate()
         except OSError as e:
-            if e.errno == os.errno.ENOENT:
+            if errno == errno.ENOENT:
```

Comment:
Finally!

Reply:
Did you mean you want to use a finally clause instead of except ?

Reply:
Clarification "Finally someone got around to fixing this!"